mock-abis/src/main/java/io/mosip/proxy/abis/service/impl/ProxyAbisInsertServiceImpl.java

```diff
@@ -68,8 +68,8 @@ public class ProxyAbisInsertServiceImpl implements ProxyAbisInsertService {
 
 	private static final Logger logger = LoggerFactory.getLogger(ProxyAbisInsertServiceImpl.class);
 
-	private static String UPLOAD_FOLDER = System.getProperty("user.dir")+"\\keystore";
-	private static String PROPERTIES_FILE = UPLOAD_FOLDER+ "\\partner.properties";
+	private static String UPLOAD_FOLDER = System.getProperty("user.dir") + "\\keystore";
```

Comment:
We should never append the file with a +

Use this approach 
'''Path currentPath = Paths.get(System.getProperty("user.dir"));
Path filePath = Paths.get(currentPath.toString(), "data", "foo.txt");
'''

Reply:
private static Path currentPath = Paths.get(System.getProperty("user.dir"));
private static Path keystoreFilePath = Paths.get(currentPath.toString(), "keystore");